go/pkg/bertymessenger/handlers.go

```diff
@@ -721,7 +721,7 @@ func (h *eventHandler) interactionFetchRelations(tx *dbWrapper, i *Interaction)
 	} else {
 		i.Conversation = conversation
 
-		if i.Conversation.Type == Conversation_MultiMemberType {
+		if i.Conversation.Type == Conversation_MultiMemberType && i.MemberPublicKey != "" {
```

Comment:
is this a normal case?

Reply:
absolutely not, this should be fetched after setting the `i.MemberPublicKey` value